static/src/stylesheets/base/_base.scss

```diff
@@ -13,6 +13,10 @@ html {
 body {
     background-color: #ffffff;
 }
+body.has-membership-access-requirement {
+    visibility: hidden !important;
```

Comment:
May be better/more-robust if you use this method of hiding: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_mixins.scss#L88